CHANGELOG.md

```diff
@@ -437,7 +425,6 @@ What's Changed
 - Fix bug sometimes causing the wrong equilibrium to be returned after optimization.
 - Improve numerical stability of perturbations and optimization.
 
-
```

Comment:
you having an OCD flare up?

Reply:
Yes. I don't understand how you all can be so inconsistent with your formatting and still sleep at night

Reply:
It was pretty consistently 2 blank lines between versions and one between sections. And for your information I don't sleep at night but thats for very different reasons.

Reply:
OK as long as there is a pattern I am happy